### ISSUE TITLE
Higher Mana Spring Totem removes Blessing of Wisdom

### ISSUE THREAD

**Issue by tedy44** [chromiecraft/chromiecraft#2812]
Just noticed that if a shaman places a mana spring totem that gives more mana than the wisdom it replaces the wisdom ... and you cannot put a new wisdom until the mana spring totem expires or it's recalled

**Comment by ishidinho** [comment]
Hey @tedy44.

First of all. Please do not use Githubs "Open blank issue". We have a form to fill out. Please use the [Get started](https://github.com/chromiecraft/chromiecraft/issues/new?assignees=&labels=&template=bug_report.yml) button to open tickets. We really need that for our work.

The issue that you describe is blizzlike. I am refering to [this source](https://wowwiki-archive.fandom.com/wiki/Mana_Spring_Totem).

> Hotfix (2009-04-23): "Greater Blessing of Wisdom and Mana Spring Totem will no longer stack as intended."

This means, that both buffs count in the same category. So, the stronger buff wins and overrides the weaker one.

> Patch 3.1.2 (2009-05-19): Greater Blessing of Wisdom now grants 92 mana per 5 seconds to prevent being overwritten by Mana Spring Totem.

To avoid this issue, Blizzard buffed GBoW at max rank to be not overridden by the totem (92 vs. 91 MP5). This should be applied to ChromieCraft but only affects the max ranks of the spell.

Long story short, this issue is blizzlike and will disappear with level 77 paladins.
